.circleci/config.yml

```diff
@@ -68,6 +77,26 @@ jobs:
       - run:
           name: Publish
           command: npm publish --access public
+      - run:
+          name: Extract version from package
+          command: |
+            echo "export PACKAGE_VERSION=$(node -p "require('./package.json').version")" >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: Login to Docker Hub
+          command: echo "$DOCKER_PASS" | docker login --username $DOCKER_USER --password-stdin
+      - run: 
+          name: Build :latest
+          command: docker build --pull --rm -f "Dockerfile" -t m365pnp/cli-microsoft365:next "." --build-arg CLI_VERSION=next
```

Comment:
Since `Dockerfile` is the default, I don't think we need to specify it here

Reply:
Nice, I didn't know this, removed.